gamemodes/core/admin/admin_cmd_lvl5.inc

```diff
@@ -0,0 +1,19 @@
+ACMD:[5]setadmin(playerid, params[])
+{
+    new target, level;
+    if(sscanf(params, "ui", target, level)) return SendSyntaxMsg(playerid, "/setadmin [player] [level]");
+    if(target == INVALID_PLAYER_ID) return SendErrorMsg(playerid, "INVALID PLAYER!");
+    if(level < 0 || level > MAX_ADMIN_LEVELS)
+    {
+        SendErrorMsgF(playerid, "Max admin level is %d", MAX_ADMIN_LEVELS);
```

Comment:
where's the return here? It would just set the admin level with the error message anyway.

Reply:
Done